ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -243,6 +243,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
+- as: aws-ipi-disc-priv-tp-f60
```

Comment:
@jinyunma @jianlinliu @gpei do we need to update the frequency to f30? as the corresponding arm jobs got removed

Reply:
seems the 4.12 is already in the extended support phase, only CVE fix will be available, please see the discussions in https://redhat-internal.slack.com/archives/GH9JHQFKN/p1728613664306739